config/initializers/audited.rb

```diff
@@ -0,0 +1,13 @@
+class KatelloAudit < Audited::Audit
+  def self.name
+    if super.start_with?("::")
+      super
+    else
+      "::#{super}"
+    end
+  end
```

Comment:
This should fix the `_reflect_on_association` errors.  @jturel does the way I added this Rails initializer look ok to you?  (I followed the instructions from Audited [here](https://github.com/collectiveidea/audited#custom-audit-model))

Reply:
for context:  This will cause `has_many` to be called correctly within the `audited` gem [here](https://github.com/collectiveidea/audited/blob/master/lib/audited/auditor.rb#L80)